pkg/ddevapp/dotddev_assets/commands/web/drupalcommands.example

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+## Description: Run Drupal commands inside the web container.
+## Usage: drupalcommands
+## Example: 'ddev drupalcommands'
+
+echo '** Check config status, get versions and important log messages, etc.'
+echo ''
+
+echo '* Configuration status'
```

Comment:
Why one asterisk vs. two? I realize it doesn’t matter all that much but you’ve got me wondering. :)

Reply:
Er, I think I see: `**` is like a top-level heading and `*` is like a subheading. If it was Markdown, `**` = `#` and `*` = `##`. But it’s not Markdown because we’re in a terminal. Ignore me!

Reply:
Yes. it's a Unix header :)